package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
```

Comment:
Because the files are no longer temporary, or ignored. Clean only makes sense if they're temporary files no?